src/marks/area.d.ts

```diff
@@ -4,30 +4,208 @@ import type {Data, MarkOptions, RenderableMark} from "../mark.js";
 import type {BinOptions, BinReducer} from "../transforms/bin.js";
 import type {StackOptions} from "../transforms/stack.js";
 
+/** Options for the area, areaX, and areaY marks. */
 export interface AreaOptions extends MarkOptions, StackOptions, CurveOptions {
+  /**
+   * The vertical position channel for the left side, typically bound to the *x*
+   * scale. When used with the **areaY** mark, it is typically equal to **x2**.
```

Comment:
The areaY mark doesn’t allow you to specify **x1**/**x2**; you can only specify **x**. This is confounding the constructor options with what happens internally (where `X2` defaults to `X1` during render); but implementation details shouldn’t be part of the public documentation.